app/assets/styles/home.css

```diff
@@ -28,19 +28,36 @@ a {
 }
 
 .header {
-  height: 70vh;
   font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
   text-align: center;
   color: #fff;
   background-color: #15154e;
 }
 
+@media only screen and (min-width: 500px) {
+  .header {
+    height: 350px;
+  }
+}
+
+@media only screen and (min-width: 800px) {
+  .header {
+    height: 470px;
+  }
+}
+
 .language-menu {
-  font-size: 1.15rem;
+  font-size: 0.7rem;
   text-align: right;
   padding: 0.5em 1em;
 }
 
+@media only screen and (min-width: 500px) {
+  .language-menu {
+    font-size: 1.15rem;
+  }
+}
+
```

Comment:
I'm also not convinced that we need to use a fixed height value. Why do you think we should use a fixed height value?

Also, for media queries to be more flexible we should define them using `ems` not `px` values. We can go back and cleanup the other media queries on the site. I'll open up a new issue for that.

Reply:
@Morban15 Seems that px vs em media queries are no longer an issue https://stackoverflow.com/questions/22228568/switching-to-em-based-media-queries. Modern browsers correctly scale px based media queries now. There are some advantages to em based media queries to benefit from scaling, but I'd say for now we can leave them as px based.